base/client.jl

```diff
@@ -71,6 +71,18 @@ function repl_cmd(cmd, out)
         ENV["OLDPWD"] = new_oldpwd
         println(out, pwd())
     else
+        @windows_only begin
+            # build correctly quoted string for cmd.exe
+            special = r".*[ &()[\]{}\^=;!'+,`~]+.*"
+            cmdstr, sep = "", ""
+            for s in cmd.exec
+                dq = ismatch(special, s) ? "\"" : ""
+                cmdstr = cmdstr * sep * dq * s * dq
+                sep = " "
+            end
+            cmd.exec = ["cmd"; "/s /c \"" * cmdstr * "\""]
```

Comment:
The idea is to use the `/S` flag to force `cmd` into the second mode of removing the surrounding `"`s and not touching the rest (re. `cmd /? )
```
    1.  If all of the following conditions are met, then quote characters
        on the command line are preserved:

        - no /S switch
        - exactly two quote characters
        - no special characters between the two quote characters,
          where special is one of: &<>()@^|
        - there are one or more whitespace characters between the
          two quote characters
        - the string between the two quote characters is the name
          of an executable file.

    2.  Otherwise, old behavior is to see if the first character is
        a quote character and if so, strip the leading character and
        remove the last quote character on the command line, preserving
        any text after the last quote character.
```
The first mode seems messy and unpredictable.